app-admin/pwman3/pwman3-0.10.0-r1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="sqlite?"
+inherit distutils-r1 eutils
+
+DESCRIPTION="A lightweight password-manager with multiple database backends"
+HOMEPAGE="https://pwman3.github.io/pwman3/"
+SRC_URI="https://github.com/pwman3/pwman3/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="mongodb mysql postgres +sqlite test"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
```

Comment:
No, now this doesn't do anything. Add it as RDEPEND and if this ebuild actually compiles anything, put them to DEPEND too.